demo/src/static/code-markdown/icon.md

```diff
@@ -0,0 +1,131 @@
+```
```

Comment:
In luma.gl we load the actual code using the ACE editor. Here we maintain separate copies of the code in markdown, which is a maintenance problem?